django-stubs/contrib/admin/widgets.pyi

```diff
@@ -38,19 +38,23 @@ class AdminFileWidget(forms.ClearableFileInput): ...
 def url_params_from_lookup_dict(lookups: Any) -> dict[str, str]: ...
 
 class ForeignKeyRawIdWidget(forms.TextInput):
-    rel: ManyToOneRel
+    rel: ManyToOneRel | ManyToManyRel
     admin_site: AdminSite
     db: str | None
     def __init__(
-        self, rel: ManyToOneRel, admin_site: AdminSite, attrs: _OptAttrs | None = ..., using: str | None = ...
+        self,
+        rel: ManyToOneRel | ManyToManyRel,
```

Comment:
This seems weird to me.
 
I think `ForeignKeyRawIdWidget` should keep accepting `ManyToOneRel`, and you should add a new `__init__` method to `ManyToManyRawIdWidget` which accepts only `ManyToManyRel`.

Reply:
I personally agree with @intgr on this one: `ForeignKeyRawIdWidget` doesn't work with `ManyToManyRel` and `ManyToManyRawIdWidget` doesn't work with `ManyToOneRel`